VKApp/VKApp/LoginViewController.swift

```diff
@@ -0,0 +1,66 @@
+// LoginViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Страница авторизации
```

Comment:
```suggestion
/// авторизации
```
Страница? У нас же не веб)